rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -83,20 +83,19 @@ SecRule REQUEST_LINE "!@rx ^(?i:(?:[a-z]{3,10}\s+(?:\w{3,7}?://[\w\-\./]*(?::\d+
 # -=[ Targets, characters and html entities ]=-
 #
 # 920120: PL1 : FILES_NAMES, FILES
-#                ['\";=] but allowed:
+#                ['\";=\x5c] but allowed:
 #                &[aAoOuUyY]uml); &[aAeEiIoOuU]circ; &[eEiIoOuUyY]acute;
 #                &[aAeEiIoOuU]grave; &[cC]cedil; &[aAnNoO]tilde; &amp; &apos;
 #
 # 920121: PL2 : FILES_NAMES, FILES
-#                ['\";=] : ' " ; = meta-characters
+#                ['\";=\x5c] : ' " ; = meta-characters
 #
 # Not supported by re2 (?<!re).
 #
 # -=[ References ]=-
-# https://www.owasp.org/index.php/ModSecurity_CRS_RuleID-960000
 # http://www.ietf.org/rfc/rfc2183.txt
 #
-SecRule FILES_NAMES|FILES "@rx (?<!&(?:[aAoOuUyY]uml)|&(?:[aAeEiIoOuU]circ)|&(?:[eEiIoOuUyY]acute)|&(?:[aAeEiIoOuU]grave)|&(?:[cC]cedil)|&(?:[aAnNoO]tilde)|&(?:amp)|&(?:apos));|['\"=]" \
+SecRule FILES_NAMES|FILES "@rx (?<!&(?:[aAoOuUyY]uml)|&(?:[aAeEiIoOuU]circ)|&(?:[eEiIoOuUyY]acute)|&(?:[aAeEiIoOuU]grave)|&(?:[cC]cedil)|&(?:[aAnNoO]tilde)|&(?:amp)|&(?:apos));|['\"=\x5c]" \
     "id:920120,\
```

Comment:
I think we need to make users aware that this rule has changed. Maybe a line in the CHANGES file to note this?